code/modules/mob/living/carbon/human/species/zombies.dm

```diff
@@ -0,0 +1,95 @@
+#define REGENERATION_DELAY 60  // After taking damage, how long it takes for automatic regeneration to begin
+
+/datum/species/zombie
+	// 1spooky
+	name = "High-Functioning Zombie"
+	name_plural = "High-Functioning Zombies"
+	icobase = 'icons/mob/human_races/r_zombie.dmi'
+	deform = 'icons/mob/human_races/r_def_zombie.dmi'
+	dies_at_threshold = TRUE
+	species_traits = list(NO_BLOOD,NOZOMBIE,NOTRANSSTING,NO_BREATHE, RADIMMUNE, NO_SCAN)
+	var/static/list/spooks = list('sound/hallucinations/growl1.ogg','sound/hallucinations/growl2.ogg','sound/hallucinations/growl3.ogg','sound/hallucinations/veryfar_noise.ogg','sound/hallucinations/wail.ogg')
+	warning_low_pressure = 50
+	hazard_low_pressure = 0
+	flesh_color = "#00FF00" // for green examine text
+	bodyflags = HAS_SKIN_COLOR
+	dietflags = DIET_CARN
+	has_organ = list(	
+		"heart" =    /obj/item/organ/internal/heart,
+		"lungs" =    /obj/item/organ/internal/lungs,
+		"liver" =    /obj/item/organ/internal/liver,
+		"kidneys" =  /obj/item/organ/internal/kidneys,
+		"brain" =    /obj/item/organ/internal/brain,
+		"appendix" = /obj/item/organ/internal/appendix,
+		"eyes" =     /obj/item/organ/internal/eyes,
+		"ears" = 	/obj/item/organ/internal/ears,
+		"zombie_infection" = /obj/item/organ/internal/zombie_infection)
+
+
+/datum/species/zombie/infectious
+	name = "Infectious Zombie"
+	mutanthands = /obj/item/zombie_hand
+	icobase = 'icons/mob/human_races/r_zombie.dmi'
+	deform = 'icons/mob/human_races/r_def_zombie.dmi'
+	armor = 20 // 120 damage to KO a zombie, which kills it
+	speedmod = 1.6
+	default_language = "Zombie"
+	language = "Zombie"
+	var/heal_rate = 1
+	var/regen_cooldown = 0
+	flesh_color = "#00FF00" // for green examine text
+
+/datum/species/zombie/infectious/handle_dna(mob/living/carbon/human/H, remove)
+	. = ..()
+
+/datum/species/zombie/infectious/spec_stun(mob/living/carbon/human/H,amount)
+	. = min(20, amount)
+
+/datum/species/zombie/infectious/spec_attacked_by(damage, damagetype = BRUTE, def_zone = null, blocked, mob/living/carbon/human/H)
```

Comment:
Question, why attach this to spec_attacked_by instead of apply_damage?

Reply:
Went with a different method for this, please look over the new code for it to make sure it’s up to standard